packages/components/radio-group/src/radio-group.css

```diff
@@ -0,0 +1,24 @@
+/* General........................................................ */
+
+:host {
+	& > fieldset {
+		border: 0 none;
+	}
+
+	& > fieldset > div {
+		padding: var(--ts-unit-quarter);
+		border-radius: 3px;
+	}
+}
+
+:host(:not([focused])) {
```

Comment:
What's the reason for this specific rule?

Reply:
No reason, it could be easy replaced with just normal hover state without any selector.
RadioGroup have the blue border shadow on hover in _ts-ui_ design. Could be 

```
:host > fieldset > div:hover {
			box-shadow: var(--ts-boxshadow-border-width-focused) var(--ts-color-blue-light);
		}
```

Reply:
Fixed